src/styles/blocks/button.scss

```diff
@@ -0,0 +1,22 @@
+.button {
+  box-sizing: border-box;
+  text-transform: uppercase;
+  color: $color-white;
+  background-color: $color-blue;
+  border: 1px solid $color-blue;
+  border-radius: 5px;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)